google/cloud/spanner/admin/database_admin_client.h

```diff
@@ -186,7 +186,7 @@ class DatabaseAdminClient {
   ///  new database.  The database ID must conform to the regular expression
   ///  `[a-z][a-z0-9_\-]*[a-z0-9]` and be between 2 and 30 characters in length.
   ///  If the database ID is a reserved word or if it contains a hyphen, the
-  ///  database ID must be enclosed in backticks (`` ` ``).
+  ///  database ID must be enclosed in backticks.
```

Comment:
Can we somehow preserve that?  Like how do we show a literal backtick in Doxygen?

Reply:
https://www.doxygen.nl/manual/markdown.html#md_codespan

"To show a literal backtick or single quote inside a code span use double backticks"

I just don't think it works. Maybe it is a bug, because all of their docs recommend the double backtick.

This is the closest I came: 
![image](https://github.com/googleapis/google-cloud-cpp/assets/23088558/0f66432c-4776-4405-97b9-b7090d94fc51)

Reply:
Blegh. Merge as-is then.  We can fix it later.

Reply:
FTR, I also tried things like `<tt>&grave;</tt>`, but they do not recognize that character. (even though they recognize `&Ograve;`, `&agrave;`, `&egrave;`, etc.